content/events/2019-amsterdam/_index.md

```diff
@@ -1,12 +1,195 @@
 ---
 title: Amsterdam
-date: 2019-09-13
+date: '2019-09-13'
+end: ''
 location:
   city: Amsterdam
-  country: The Nederlands
+  country: The Netherlands
   url: https://dezwijger.nl
   venue: Pakhuis De Zwijger
+map: amsterdam-map.png
+sched_slug: kcdams2019
 social:
   facebook: https://facebook.com/cloudnativeams
   twitter: https://twitter.com/cloudnativeams
+sponsors:
+- level: Partners
+  orgs:
+  - name: Container Solutions
+    logo: sponsors/container-solutions.jpg
+    url: https://www.container-solutions.com/
+  - name: Fullstaq
+    logo: sponsors/fullstaq.png
+    url: https://fullstaq.com/
+- level: Coffee & Drinks
+  orgs:
+  - name: Sentia
+    logo: sponsors/sentia.png
+    url: https://www.sentia.com/
+  - name: SUSE
+    logo: sponsors/suse.jpg
+    url: https://www.suse.com/
+- level: gold
+  orgs:
+  - name: Rancher
+    logo: sponsors/rancher.png
+    url: https://rancher.com/
+  - name: Microsoft
+    logo: sponsors/microsoft.png
+    url: https://microsoft.com
+  - name: Lightbend
+    logo: sponsors/lightbend.png
+    url: https://www.lightbend.com/
+  - name: ABN AMRO
+    logo: sponsors/abn-amro.svg
+    url: https://www.abnamro.com/
+  - name: Lunatech & D2IQ
+    logo: sponsors/lunatech.png
+    url: https://www.lunatech.com/
+  - name: CloudBees
+    logo: sponsors/cloudbees.png
+    url: https://www.cloudbees.com/
+  - name: Vamp
+    logo: sponsors/vamp.jpeg
+    url: https://www.vamp.io/
+  - name: Netapp
+    logo: sponsors/netapp.jpg
```

Comment:
Please use an SVG for all images. They're all at landscape.cncf.io

Reply:
Addressed for most sponsors (a few are not CNCF members). Should look better.